triggers/README.md

```diff
@@ -1,21 +1,55 @@
-# Create a build trigger for a GitHub repository
+# Cloud Build triggers for a GitHub repository
 
-We use Google Cloud Build (GCB) triggers to test the applications.
+We use Google Cloud Build (GCB) triggers to test applications.
 
-*   Run the following command to create a trigger for your Kubernetes
+For more information, see the
+[`gcloud alpha builds triggers`](https://cloud.google.com/sdk/gcloud/reference/alpha/builds/triggers/)
+commands.
+
+## Create or update a trigger
```

Comment:
In this section we should also provide guidance how to create the triggers. Should it happen manually and then get exported? How to check for the installationId? How should one use the trigger id and name?